hera_opm/mf_tools.py

```diff
@@ -664,11 +664,14 @@ def build_lstbin_makeflow_from_config(config_file, mf_name=None, work_dir=None):
             outfiles = make_outfile_name('lstbin_outfile_{}'.format(output_file_index), 'LSTBIN', pol_list)
 
             # get args list for lst-binning step
-            args = [get_config_entry(config, "LSTBIN_OPTS", a, required=False) for a in lstbin_args]
-
-            # flatten list
-            args = [item for sublist in args for item in sublist]
-
+            _args = [get_config_entry(config, "LSTBIN_OPTS", a, required=False) for a in lstbin_args]
```

Comment:
Also, when you get `lstbin_args` above, you have `required=False`. If that's the case, then `lstbin_args` could be `None`. We should put the argument processing into an `if lstbin_args is not None:` block, or we could make the `lstbin_args` config option required. Up to you.